spec/models/element_spec.rb

```diff
@@ -47,9 +47,12 @@ module Alchemy
         expect(subject.contents.count).to eq(element.contents.count)
       end
 
-      it "should create a new record with all attributes of source except given differences" do
-        copy = Element.copy(element, {name: 'foobar'})
-        expect(copy.name).to eq('foobar')
+      context 'with differences' do
+        subject(:copy) { Element.copy(element, {name: 'foobar'}) }
+
+        it "should create a new record with all attributes of source except given differences" do
+          expect(copy.name).to eq('foobar')
```

Comment:
I just cleaned the test code and did not changed it's implementation nor wording inside this branch, so you can ignore this and discuss this in an extra issue/branch.